clang/lib/Sema/SemaType.cpp

```diff
@@ -7941,6 +7942,36 @@ static bool handleFunctionTypeAttr(TypeProcessingState &state, ParsedAttr &attr,
     return true;
   }
 
+  if (attr.getKind() == ParsedAttr::AT_CFISalt) {
+    if (attr.getNumArgs() != 1)
+      return true;
+
+    StringRef Argument;
+    if (!S.checkStringLiteralArgumentAttr(attr, 0, Argument))
+      return false;
```

Comment:
I think this should be `return true`; returning `false` basically says "try to slide this attribute around to something else until it makes sense" but in this case the attribute is wrong and won't make sense anywhere.

Reply:
Oh yeah...I'm not sure why I missed that. Fixed.